dll/shellext/netshell/lanstatusui.h

```diff
@@ -0,0 +1,62 @@
+
+/// CLSID
+/// HKEY_LOCAL_MACHINE\SOFTWARE\Classes\CLSID\{7007ACCF-3202-11D1-AAD2-00805FC1270E}
+// IID B722BCCB-4E68-101B-A2BC-00AA00404770
+
+#define WM_SHOWSTATUSDLG    (WM_USER+10)
+
+typedef struct tagNotificationItem
+{
+    struct tagNotificationItem *pNext;
+    CLSID guidItem;
+    UINT uID;
+    HWND hwndDlg;
+    INetConnection *pNet;
+} NOTIFICATION_ITEM;
```

Comment:
I know you only moved this, but consider also renaming it to not have the same name as the thing in winlogon.
Also, same comment as the other struct, giving this a dtor and using `new` / `delete` to allocate / cleanup will allow it to clean up `pNext` (and `pNet` if that is needed).